portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,155 @@
+                           in new york city (brooklyn, to be exact) and it's honestly my favorite place in the world!
+                           i actually have a bunch of goals that i hope to accomplish in the future, many from varying fields,
+                           so i hope that amidst my engineering career, i am able to pursue careers on the side! for instance,
+                           i would love to be a social media influencer, graphic designer or even a film director! another fact about me? i love music! 
+                           my favorite band is the 1975, and my favorite artist right now is harry styles. i've actually 
+                           been a big fan of both since i was thirteen and was supposed to see them
+                           this year until the pandemic started :(
+                        </p>
+                    </div>
+                    <div class="portfolio-img">
+                        <img id="biomal" src="/images/mally3.jpg">
+                    </div>
+                </div>
+
+                <div class="portfolio-item">                    
+                    <div class="portfolio-description">
+                        <img src = "images/hobbies.png">
+                        <p>
+                        my hobbies include thrifting (i'm very into sustainability), traveling, going to the beach, shopping,
+                        listening to music and filming youtube videos! i actually have a youtube channel that could be
+                        found <strong><a href="https://www.youtube.com/channel/UCp8Eay8DqjQq3J0p0J3jwOw">here.</a></strong>
+                        though it's very new, i definitely want to film videos about fashion, lifestyle and productivity! i'm also extremely
+                        into art, marketing and film as i said before, and i have been using video and photo editing softwares since i was younger.
+                        i love all of my hobbies and i hope that one day i'll be able to use them in order to make the world a better place!!
+                        </p>
+                    </div>
+                    <div class="portfolio-img">
+                        <img src="images/camera.JPG">
+                    </div>
+                </div>
+        </section>
+<!-- PROJECTS -->
+        <section class="projects" id="projects">
+            <div class="container">
+                <div class="section-heading">
+                    <img id = "projectspic" src = "images/projects.png">
+                </div>
+
+                <div class="portfolio-item">                    
+                    <div class="portfolio-description">
+                        <img src="images/modelrocket.png">
+                        <p>
+                            • cooperate with b-above: aviation student association and bingspace members (15+) in creating a rocket for the seds competition, combining
```

Comment:
Instead of adding these bullet points in as a character you should consider using a `<ul>` element. (stands for Unordered List). This is exactly what it is designed for! You may have to play with the CSS to get it looking exactly the same but I think it would be worth it.
```
<ul>
  <li>cooperate with...</li>
  <li>research student... </li>
  <li>helped perform... </li>
</ul>
```

Reply:
Done! Thanks for the tip! I learned something new today :)